data-gen/sdc/csvReviews-sdc.js

```diff
@@ -6,8 +6,8 @@ const csvHeader = `id_user,id_game,is_recommended,hours_on_record,hours_at_revie
 
 writeReviewsData.write(csvHeader, 'utf-8');
 
-let generateRandomNum = (min = 1, max, float) => {
-  return float ? Math.floor((Math.random() * (max - min) + 1000)) / 10 : Math.floor((Math.random() * max) + min)
+let generateRandomNum = (min = 1, max, float, precision = 10) => {
```

Comment:
Not entirely sure I understand this part - if you changed the precision, wouldn't you have to change the "1000" in there, too? I may be totally missing something, but just make sure that if you change the "precision" without changing any hardcoded values, it still works as expected. And if it doesn't, adjust the hardcoded values to have the proper relationship to the "precision" variable.

Reply:
On line 10, I changed the division with the precision variable

> return float ? Math.floor((Math.random() * (max - min) + 1000)) / precision : Math.floor((Math.random() * max) + min)

there is a flaw on this function tho, the max-min doesn't seem to really matter, the variable precision at this point just determines how many decimal places I want the returned value to be, (tenths, hundredths, thousanths..) but yeah the ' /10' got changed to ' / precision' so now if there is a precision included when calling this helper function, it should change the expected output